module_9/task3.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+$numbers = [];
+
+for ($i = 0; $i < rand(3, 20); $i++) {
```

Comment:
Be carefully with functions in comparitions with `for` function. Function will have been ran with every iteration and you will never have clear result. You should implement a variable before cicle and use variable in comparition